Example/ParalayoutDemo.xcodeproj/project.pbxproj

```diff
@@ -314,7 +314,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.6;
```

Comment:
any particular reason for `.6` vs `.0`?

Reply:
cause that's what Xcode auto-populated when I selected "iOS 15" from the drop down 😂 🤷‍♂️

Reply:
😅 that's hit me a few times too.